.github/workflows/continuous-deployment-dev.yml

```diff
@@ -23,8 +23,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Set API URL
```

Comment:
Not sure if i understand this correctly, but isnt the default input set to `https://api.dev.cloud.dpst.dev/api/v1` ? 
Why do we need to check again if its set ?

Reply:
it's not set when we do a push to main, only on workflow dispatch

Reply:
Okay, way do we run this action on main then ? 🤔 Couldnt we just keep this as a remote workflow dispatch. I think we are currently running everything twice on main, right ?